### DISCUSSION TITLE
local.ERROR: Disabled, not allowed to import.

### DISCUSSION THREAD

**Discussion by jimmisavage** [firefly-iii/firefly-iii#8992] [General]
I have recently installed firefly iii along with the importer and cron to update overnight from my gocardless account.

The cron is flagging an error when trying to update - local.ERROR: Disabled, not allowed to import. {"exception":"[object] (App\\Exceptions\\ImporterErrorException(code: 0): Disabled, not allowed to import. at /var/www/html/app/Http/Controllers/AutoImportController.php:48)

Anyone know what's going on here?

**Comment by JC5** [comment]
https://github.com/firefly-iii/data-importer/blob/main/.env.example#L112

**Reply by jimmisavage** [reply]
Oh! how have i missed that? Thanks for the help :)
